tools/notifyServicelogEvent.c

```diff
@@ -177,18 +169,21 @@ main(int argc, char *argv[])
     }
 
   done:
+    g_strfreev(processed_args);
+    pcmk__free_arg_context(context);
+
     if (event != NULL) {
         servicelog_event_free(event);
     }
 
     if (slog != NULL) {
         servicelog_close(slog);
     }
-
     closelog();
 
     if (exit_code == CRM_EX_OK) {
         exit_code = pcmk_rc2exitc(rc);
     }
```

Comment:
I suspect we don't need to bother checking rc here, I think exit_code should be sufficient

Reply:
We've got the rc right at the end from `pcmk__attrd_api_update()`

Reply:
Though as I noted in an earlier commit, apparently the original version doesn't throw an error code when attrd_update fails

Reply:
If you handle rc != pcmk_rc_ok after pcmk__attrd_api_update fails by setting exit_code, I think you've covered it all and can drop this block right here.  I don't see any other ways to get an error code into rc.